docs/detections/detections-ui-exceptions.asciidoc

```diff
@@ -1,35 +1,28 @@
 [[detections-ui-exceptions]]
 == Rule exceptions
 
-Rule exceptions can be associated with detection and endpoint rules to prevent those rules from generating alerts under specific circumstances. Exceptions are made up of three main components: exception items, default rule lists, and shared exception lists.
+Rule exceptions can be associated with detection and endpoint rules to prevent those rules from generating alerts under specific circumstances. You can use exceptions to reduce the number of false positives, and to prevent trusted processes and network activity from generating unnecessary alerts.
+
+When creating exceptions, you can assign them to <<rule-exceptions-intro,individual rules>> or share them among <<shared-exception-list-intro,multiple rules>>.
 
 [float]
-[[exception-item-intro]]
-=== Exception items
+[[rule-exceptions-intro]]
+=== Exceptions for individual rules
 
-Exception items, also referred to as _exceptions_, contain the source event conditions that determine when alerts are not generated. You can use exceptions to reduce the number of false positives, and to prevent trusted processes and network activity from generating unnecessary alerts. 
+Exceptions, also referred to as _exception items_, contain the source event conditions that determine when alerts should or should not be generated. 
 
-Rules can have multiple exceptions and exceptions can apply to multiple rules. Refer to <<add-exceptions>> to learn more about adding exceptions to rules. 
+You can create exceptions that apply exclusively to a single rule. These types of exceptions cannot be used by other rules and you must manage them from the rule’s details page. To learn more about creating and managing these types of exceptions, refer to <<add-exceptions>>.
```

Comment:
To avoid repeating "these types of exceptions"?
```suggestion
You can create exceptions that apply exclusively to a single rule. These types of exceptions can't be used by other rules, and you must manage them from the rule’s details page. To learn more about creating and managing single-rule exceptions, refer to <<add-exceptions>>.
```

Reply:
This is a google alternative. Thanks for the suggestion.